javascript/index.js

```diff
@@ -0,0 +1,127 @@
+        }
+    }
+    // error message remove
+    else if(isValidEmail(inputEmail.value)) {
+        if(inputBoxEmail.lastElementChild !== inputEmail) {
+            inputBoxEmail.lastElementChild.remove();
+            inputEmail.classList.remove('error-border');
+        }
+    }
+}
+// focus-out password
+function eventFocusOutPassword() {
+    // no password value
+    if(inputPassword.value === "") {
+        if(inputBoxPassword.lastElementChild === inputPassword) {
+            const nothingInput = document.createElement("p");
+            nothingInput.textContent = "비밀번호를 입력해주세요.";
+            inputBoxPassword.append(nothingInput);
+
+            inputPassword.classList.add('error-border');
+            nothingInput.classList.add('error-text');
+        } else {
+            inputBoxPassword.lastElementChild.textContent = "비밀번호를 입력해주세요.";
+        }
+    }
+    else {
+        if(inputBoxPassword.lastElementChild !== inputPassword) {
+            inputBoxPassword.lastElementChild.remove();
+            inputPassword.classList.remove('error-border');
+        }
+    }
+}
+// login button click
+function eventClickBtn() {
+    if(inputEmail.value === "test@codeit.com" && inputPassword.value === "codeit101") {
+        window.location.href = "/folder"
+    }
+    if(inputEmail.value !== "test@codeit.com") {
+        if(inputBoxEmail.lastElementChild === inputEmail) {
+            const invalidInput = document.createElement("p");
+            invalidInput.textContent = "아이디를 확인해주세요.";
+            inputBoxEmail.append(invalidInput);
+
+            inputEmail.classList.add('error-border');
+            invalidInput.classList.add('error-text');
+        } else {
+            inputBoxEmail.lastElementChild.textContent = "아이디를 확인해주세요.";
+        }
+    }
+    if(inputPassword.value !== "codeit101") {
+        if(inputBoxPassword.lastElementChild === inputPassword) {
+            const nothingInput = document.createElement("p");
+            nothingInput.textContent = "비밀번호를 확인해주세요.";
+            inputBoxPassword.append(nothingInput);
+
+            inputPassword.classList.add('error-border');
+            nothingInput.classList.add('error-text');
+        } else {
+            inputBoxPassword.lastElementChild.textContent = "비밀번호를 확인해주세요.";
+        }
+    }
+}
+// press Enter
+function eventPressEnter(e) {
+    if(e.keyCode == 13) {
+        eventClickBtn();
+    }
+}
+
+// press eye-icon
+function eventClickEye() {
+    if(inputPassword.type === "password") {
+        inputPassword.type = "text";
+        eyeImg.src = "./images/eye-on.png";
+    } else {
+        inputPassword.type = "password";
+        eyeImg.src = "./images/eye-off.png";
+    }
+}
+
+inputEmail.addEventListener('focusout', eventFocusOutEmail);
+inputPassword.addEventListener('focusout', eventFocusOutPassword);
+loginButton.addEventListener('click', eventClickBtn);
+document.addEventListener('keyup', eventPressEnter);
```

Comment:
c: 만약 이벤트 핸들러 컨벤션을 event[event name][target] 으로 맞췄다면 다음과 같이 네이밍하는게 더 적절할 것 같아요
```suggestion
document.addEventListener('keyup', eventKeyUp);
```